resources/custom/Homestead.yaml

```diff
@@ -0,0 +1,113 @@
+---
+ip: "192.168.10.10"
+memory: 4096
+cpus: 2
+provider: virtualbox
+elasticsearch: 6
+
+authorize: ~/.ssh/id_rsa.pub
+
+keys:
+    - ~/.ssh/id_rsa
+
+folders:
+    - map: "{codeRepositoryPath}"
+      to: /home/vagrant/code
+      type: "nfs"
+    - map: "{vendorPath}"
+      to: /home/vagrant/vendor
+      type: "nfs"
+    - map: "{vendorPath}/../files/config"
+      to: /home/vagrant/config
+      type: "nfs"
+
+sites:
+    # API 3
+    - map: api3.test
+      to: /home/vagrant/code/api3/public
+      php: "7.1"
+
+    # Auth Service
+    - map: auth-service.test
+      to: /home/vagrant/code/auth-service/public
+      php: "7.1"
+
+    # Contractors Tool
+    - map: contractors-tool.test
+      to: /home/vagrant/code/contractors-tool/public
+      php: "7.1"
+
+    # Dashboard API
```

Comment:
Is this in use?

Reply:
I personally use it. It's the simple dashboard I've created to grab and regroup the issues from Sentry by the most common ones